sentry/src/main/java/io/sentry/UncaughtExceptionHandlerIntegration.java

```diff
@@ -147,6 +147,11 @@ static Throwable getUnhandledThrowable(
     final Mechanism mechanism = new Mechanism();
     mechanism.setHandled(false);
     mechanism.setType("UncaughtExceptionHandler");
+
+    // probably should be Android only?
+    if (thrown instanceof RuntimeException) {
```

Comment:
A `RuntimeException` by itself isn't an exception group, there's suppressed exceptions which we have an issue for (https://github.com/getsentry/sentry-java/issues/542) but I'm guessing that has little to do with the issue we're trying to fix here.